From f0c5a8c276b055a90a771faf93460259299b424f Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Wed, 21 Jun 2023 09:19:47 +0800
Subject: [PATCH] [riscv32]Skip regress-crbug-781583

the number of cascading generators is too low. It can't produce rangeerror on riscv32.

Change-Id: Id5e784d941e3b0debb75fda6193ea8d096ddd6ca
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/4630554
Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
Reviewed-by: Michael Achenbach <machenbach@chromium.org>
Cr-Commit-Position: refs/heads/main@{#88473}
---
 test/mjsunit/mjsunit.status | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mjsunit/mjsunit.status b/test/mjsunit/mjsunit.status
index 0175b0f0c72..863cf96042f 100644
--- a/test/mjsunit/mjsunit.status
+++ b/test/mjsunit/mjsunit.status
@@ -973,6 +973,7 @@
 
 [ 'arch == riscv32' , {
 'wasm/compare-exchange64-stress':[SKIP],
+'regress/regress-crbug-781583':[SKIP],
 }], # 'arch == riscv32'
 
 ##############################################################################
-- 
2.35.1

